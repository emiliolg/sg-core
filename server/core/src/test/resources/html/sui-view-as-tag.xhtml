<sui-view sui-view="tekgenesis.service.html.Views">
    <title>Page Title</title>
    <meta name="description" content="description"/>
    <meta charset="utf-8"/>
</sui-view>