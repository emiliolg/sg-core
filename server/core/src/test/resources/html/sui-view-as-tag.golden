<!DOCTYPE html>

    <title>Page Title</title>
    <meta name="description" content="description"></meta>
    <meta charset="utf-8"></meta>
